examples/flask/server.py

```diff
@@ -44,12 +52,72 @@
 """
 
 import argparse
+import os
 
 import flask
 from flask_cors import CORS
 
-from deepsparse import compile_model
-from deepsparse.utils import arrays_to_bytes, bytes_to_arrays
+from deepsparse import Scheduler, compile_model
+from deepsparse.utils import arrays_to_bytes, bytes_to_arrays, log_init
+
+
+_LOGGER = log_init(os.path.basename(__file__))
+
+
+def engine_flask_server(
+    model_path: str,
+    batch_size: int = 1,
+    num_cores: int = None,
+    num_sockets: int = None,
+    scheduler: Scheduler = None,
```

Comment:
Shouldn't this default to the multi-stream scheduler?

Reply:
I believe this should be handled by the `compile_model` function.
Was going off of the defaults [here](https://github.com/neuralmagic/deepsparse/blob/main/src/deepsparse/engine.py#L548).

Reply:
are you saying that for a client/server scenario, we want to instead default to multi-stream?

Reply:
that makes sense to me, so going with that.

Reply:
Yeah, that was the general idea for multi-stream.  Unless @mgoin or anyone else has objections.  It should be fine either way for an example.

Reply:
makes sense. thanks for the catch @bnellnm